.travis.yml

```diff
@@ -3,4 +3,17 @@ import:
   - Alfresco/alfresco-process-tools:.travis.pre-commit.yml@master
   - Alfresco/alfresco-process-tools:.travis.helm-docs_install.yml@master
   - Alfresco/alfresco-process-tools:.travis.kubepug_install.yml@master
-  - Alfresco/alfresco-process-scripts:slack_apa_channel_notification.yml@master
+
+notifications:
+  slack:
+    rooms:
+      secure: "G4Tlvx6kP5PT5hOCfpHacee87OvSIRZE/uNe7TlymyOmTeJma+76RVS+dsEuT9Q6w/Kc3N/WoBjm0j7eq9fl08RvMMkmdNjb1fJ205YHHtvnul4xmzkb4C2457XN61OYL9pJysUZozYiajAj2PiX4EnQhW04PsZsxafh+Ws4yIIkW2KnA7AsomusqTWhcmNlWuyAQVCD8gB6rWZnJz2G1wrfKqfgqyb7W6aWFZe4DRp8uj0k91vgD8DPFoPvTEuw9EtikkO4SakbCJXEDKqKx9gD5/hEAsCECBhE9ZHeQERzXSpN93AVH1kvpMd/a+TfMbELdkdvPLioNdM2/hzSFuu4RQS+0SOpnMZNnnXfE4+QpMAgESQsfRIUWepJdGsOxVUUpWNylXnDeW7lvwKUzUI/pWnwwDVR87WkUfB+1RKGhPJIvRZd7ZljihbJTzFuoT6eYnlcRsRTYUrpZaxQGV6ZUt8BEaRM3Ow9iZmv9AMp7YGdM3j8DDVEKAojQPpzbPhz8NuSoAplB6SSoGwB/qdAdYmyjwz+eQFypyrGfTzVdXFDVS3Rn/VRUnRhoXdpPVfDClcGTrAaEMoOvyCtA79QqcHdN0qcHPJdE6D6Gme239m+A4kYNwzFXCMmCbT8IWgv+0CC0UaAny2rCb0G95oTNgYEpcAtOR3/G5Ga0iM="
+    on_pull_requests: false
+    on_failure: always
+    on_success: never
```

Comment:
can you check if any of these options are redundant or not?
for example is `on_success: never` redundant or needs to be specified ?